libs/react-renterd/src/bus.ts

```diff
@@ -838,3 +840,161 @@ export function useMetricsWallet(
 //   hostKey: string
 //   origin: string
 // }
+
+// multipart
+
+export type MultipartUploadCreatePayload = {
+  path: string
+  bucket: string
+  key?: string /// defaults to key:0000000000000000000000000000000000000000000000000000000000000000
```

Comment:
You don't need to set this anymore. Set `generateKey` to `true` instead to get a random key by default.

Reply:
So I understand, previously I think omitting this key would generate a key, now I pass generateKey: true? But there is still an option to pass a key?

Reply:
Previously omitting the key would use the noop key `key:000000000000000000000000000000000000000000000000000000000000000` which disables the pre-shading encryption.
Since we want that encryption, you need to specify `generateKey: true` for a key to be generated.